App/Navigation/OnboardingNavigation.js

```diff
@@ -1,78 +1,51 @@
 import { StackNavigator } from 'react-navigation'
 import OnboardingScreen from '../Containers/OnboardingScreen'
+import { Colors } from '../Themes'
 import LoginScreen from '../Containers/LoginScreen'
 import {getPhoto} from '../Services/PhotoUtils'
 import Actions from '../Redux/TextileRedux'
 import styles from './Styles/NavigationStyles'
+import ostyles, {buttonColor} from '../Containers/Styles/OnboardingScreenStyle'
 
 export const params1 = {
   header: 'Welcome to the Textile Beta!',
   message: 'Thanks very much for agreeing to test the app and provide feedback on this very early draft of our brand new technology. Right now, Textile is all about photos, soon it will be about how you control and own all your personal data.',
   buttonTitle: 'OK',
   onButtonPress: (navigate) => {
     return () => {
-      navigate('Onboarding1', params2)
+      navigate('OnboardingInfo', params2)
     }
   }
 }
 
 const params2 = {
-  header: 'Today we’re sharing our backup and sync tools.',
-  message: 'While we don\'t recommend you delete your photos until our full release, right now you can use our system to upload backups of your private photos to decentralized servers around the world and then retrieve those photos at any time. You can also use our desktop tool to magically sync all your photos right to your computer.',
-  buttonTitle: 'GOT IT',
-  onButtonPress: (navigate) => {
-    return () => {
-      navigate('Onboarding2', params3)
-    }
-  }
-}
-
-const params3 = {
-  header: 'We’ve got big plans…',
-  message: 'We want to test out our infrastructure for moving, storing, and accessing private photos today. Next, we hope to provide you with some really exciting ways to share photos and interact with your friends and family on the same system. Beyond that, we want to give you secure tools to regain control over all your data.',
-  buttonTitle: 'COOL',
-  onButtonPress: (navigate) => {
-    return () => {
-      navigate('Onboarding3', params5)
-    }
-  }
-}
-
-const params5 = {
-  header: 'Private, secure, fast, and distributed.',
-  message: 'Right now, the Textile mobile app quietly listens for new photos you take with your camera. Next, it securely encrypts those photos in a way that only you (not even our servers) can ever open them. It then stores those files on servers using a system called, IPFS. All the while, it generates a private wallet that you can keep and eventually move to other systems where you keep control of all your data.',
-  buttonTitle: 'OK',
-  onButtonPress: (navigate) => {
-    return () => {
-      navigate('Onboarding4', params6)
-    }
-  }
-}
-
-const params6 = {
   header: 'You\'re helping us improve.',
-  message: 'As a valued Beta Tester, we want to know how, when, and why you are using the app. We anonymously collect data, including crashes, screen interactions, and feedback. If you want to provide direct feedback, please feel free to email us with you thoughts, comments, or ideas.',
+  message: (
+    <View>
```

Comment:
`OnboardingScreen` currently is wrapping the value if this `message` prop in a pair of `<Text>` tags. So in this case, it's rendering `<Text><View>...</View></Text>`, which might work, but is not right. I'd just update `OnboardingScreen` to expect that the value of message is a component without assuming what it is exactly. Know what I mean?

Reply:
fixed